ingestion_server/test/integration_test.py

```diff
@@ -267,6 +267,30 @@ def _promote(self, model, suffix="integration", alias=None):
         es = self._get_es()
         assert list(es.indices.get(index=alias).keys())[0] == f"{model}-{suffix}"
 
+    def _delete_index(self, model, suffix="integration", alias=None):
+        req = {
+            "model": model,
+            "action": "DELETE_INDEX",
+            "callback_url": bottle_url,
+        }
+        if alias is None:
+            req |= {"index_suffix": suffix}
+        else:
+            req |= {
+                "alias": alias,
+                "force_delete": True,
```

Comment:
Can you add a test for when force_delete is not passed and also when it is not True?

Reply:
Added these tests cases in fb4fa02 (and got them to work 6 commits later 😆).